yarn.lock

```diff
@@ -1,105 +1,152 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
+# This file is generated by running "yarn install" inside your project.
```

Comment:
Don't overwrite the lockfile. I don't think our Yarn V1 support lockfile in this format.